compiler/src/main/java/org/robovm/compiler/config/Config.java

```diff
@@ -1361,7 +1396,12 @@ public void addPluginArgument(String argName) {
             }
             config.pluginArguments.add(argName);
         }
-
+        
+        // TODO: Are we disobeying any rules by allowing config to be fetched?  All I want is the targetBinary.
```

Comment:
Do you need some help?  I just found the discussion in github - had looked several times but it wasn't easy to find!  I could rebase this branch to your latest master and then also fix this.